spec/src/main/asciidoc/method-query.asciidoc

```diff
@@ -108,6 +108,16 @@ Absence of an operator implies the equality condition.
 
 The conditions belonging to the restriction determine the parameters of the method, as specified below in <<Query by Method Name Conditions>>.
 
+[WARNING]
+====
+When using NoSQL databases, the support for restrictions varies depending on the database type:
+
+- **Key-Value Databases**: Support for the equals restriction and the `Not` restriction is required for the `Id` attribute. There is no requirement to support other types of restrictions, the `And` and `Or` keywords, or restrictions on other entity attributes. Queries in key-value databases are typically limited to operations using the `Id` attribute only.
```

Comment:
@njr-11 Key-value only supports `equals`. There is no way to support `Not`.

it will work like a `Map` in Java, thus, there is only the `get` to read information.